frameworks/Clojure/donkey/src/hello/handler.clj

```diff
@@ -0,0 +1,49 @@
+(ns hello.handler
+  (:require [com.appsflyer.donkey.core :as donkey-core]
+            [com.appsflyer.donkey.server :as donkey-server]
+            [com.appsflyer.donkey.middleware.json :as json])
+  (:gen-class)
+  (:import (io.vertx.core.impl.cpu CpuCoreSensor)))
+
+(def ^:private ^:const json-response
+  {:status  200
+   :headers {"content-type" "application/json"}
+   :body    {"message" "Hello, World!"}})
+
+(defn- json-handler [_ res _]
+  (res json-response))
```

Comment:
`json-response` is a const that's not being reconstructed on every request. See the test rules for json: https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#json-serialization

> i. For each request, an object mapping the key message to Hello, World! must be instantiated.

Reply:
Updated as requested.